packages/flutter/lib/src/painting/notched_shapes.dart

```diff
@@ -37,10 +37,15 @@ abstract class NotchedShape {
 ///
 ///  * [CircleBorder], a [ShapeBorder] that describes a circle.
 class CircularNotchedRectangle extends NotchedShape {
+  /// A convex notch shape. It's concave by default.
```

Comment:
Nit: Dart docs specify the first sentence should be its own paragraph. https://dart.dev/guides/language/effective-dart/documentation#do-separate-the-first-sentence-of-a-doc-comment-into-its-own-paragraph

Reply:
A trailing line like that:

```dart
/// A convex notch shape. It's concave by default.
///
final bool convex;
```

? Ok.